app/controllers/lettings_logs_controller.rb

```diff
@@ -11,20 +13,20 @@ class LettingsLogsController < LogsController
   before_action :redirect_if_bulk_upload_resolved, only: [:index]
 
   def index
-    respond_to do |format|
-      format.html do
```

Comment:
I got rid of this format stuff for this method because we never do any other format than the default. Someone tell me if any of that is not quite true.

Reply:
I guess there was also `format.csv` in the past, so it's probably fine to remove now